utils/leafblower.py

```diff
@@ -383,7 +383,10 @@ def find_functions(self):
 
                 register_used = instruction.getOpObjects(
                     reference.getOperandIndex())
-                register_index = registers.index(register_used[0].toString())
+                try:
+                    register_index = registers.index(register_used[0].toString())
+                except ValueError:
+                    print("List does not contain value")
```

Comment:
@sparrow-labz Thanks for submitting the pull request. I don't think I've ever run across this error so I'm wondering if it's an architecture problem? (arm vs x86 vs etc) Could you give a little background on how you ran across this and maybe provide the binary? Regardless, this looks like it should still throw an exception since the ValueError exception would result in `register_index` not being declared but is used in the `FormatFunction` instantiation a few lines below.

Reply:
I ran leafblower on an arm bootloader binary for an older Samsung phone, from the A series. I'll try to find the binary, and add it.